app/src/test/java/ru/yandex/praktikumchatapp/ChatViewModelTest.kt

```diff
@@ -32,12 +41,26 @@ class ChatViewModelTest {
     @Test
     fun `send message should update messages with MyMessage`() = runTest {
         val message = Message.MyMessage("TestMessage")
-
+        viewModel.sendMyMessage("TestMessage")
```

Comment:
```suggestion
        viewModel.sendMyMessage(message.text)
```
Лучше так написать, чтобы не копировать содержимое сообщения в двух местах